imports/ui_3/views/layouts/main.js

```diff
@@ -139,6 +141,20 @@ Template.Main_layout.onRendered(function() {
   });*/
 });
 
+const CHOPAT_CHARS = 600;
+
+Template.registerHelper('chop', function chop(text, chars = CHOPAT_CHARS) {
+  if (isNaN(chars)) chars = CHOPAT_CHARS; // Bypassing Blaze auto last params in helper if chars is not set
+  if (text.length <= chars) return text;
+  const textPart = text.substr(0, chars);
+  return textPart + '... ';
+});
+
+Template.registerHelper('showmoreButton', function showmoreButton(chars = CHOPAT_CHARS) {
+  if (!this.text || (this.text && this.text.length <= chars)) return;
```

Comment:
A showMoreButton-nak is at kene adni a text parametert, mert ha o this.text-et hasznal, akkor csakis "text" nevu dolgokat tudunk chop=polni, es a {{chop title 20}} pl mar nem fog mukodni (ha akarnank ilyet)